go.mod

```diff
@@ -7,3 +7,5 @@ require (
 	golang.org/x/crypto v0.10.0
 	golang.org/x/sys v0.9.0
 )
+
+replace github.com/google/go-pkcs11 => /usr/local/google/home/lforges/go-pkcs11
```

Comment:
It's best to avoid committing local replace actions.

This would break everyone else using this code.